### PR TITLE
(No ticket) Wharton QA comments

### PR DESCRIPTION
**Pull request by wendysuarez**
QA comments [Wharton BA](https://www.figma.com/file/JFA4P2HsDcZKAfTmBee9mw/WBA-Review-Halfises---06-19-2020?node-id=9%3A10)
QA comments [Wharton HR](https://www.figma.com/file/Bhk5AqSC3KJLUlVhNwyWl3/WHR-Review-Halfsies---6-19-2020?node-id=1%3A47)

### CODE REVIEW

File: styles/src/modules/wharton/business_analytics/sections/_case_studies.scss [hunk 1]
```diff
@@ -8,7 +8,10 @@
     color: $em-color-light;
     font-weight: $wharton-font-weight-regular;
     height: 100%;
+    margin: .5rem;
```
Comment (jesseocon) [inline styles/src/modules/wharton/business_analytics/sections/_case_studies.scss:11]: I think it is safe that we make all case studies have the standard bootstrap 15px on mobile. @bocha should we make this a change in the common styles?

File: styles/src/images/wharton/business_analytics/emeritus-phone-icon.svg [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
...2 2.572.034.859.801 1.689 1.644 2.487 2.501.608.652.608 1.66 0 2.312-.424.48-.895.917-1.344 1.375-.04.039-.084.072-.159.14 1.199 2.158 2.988 3.933 5.16 5.12l1.072-1.078c.263-.31.593-.555.965-.72.598-.259 1.293-.136 1.764.312.9.851 1.776 1.727 2.63 2.626.6.64.62 1.625.046 2.288-.54.634-1.14 1.234-1.745 1.815-.43.394-.98.638-1.563.693-.042.01-.083.022-.123.036l-.712.01zm.268-1.083c.601.033 1.187-.196 1.604-.629.406-.422.82-.835 1.23-1.253.054-.053.104-.109.15-.168.141-.141.219-.333.215-.532-.004-.2-.09-.389-.236-.524-.33-.355-.68-.691-1.023-1.035-.447-.444-.888-.89-1.34-1.327-.132-.149-.32-.236-.52-.241-.2-.006-.393.071-.534.212-.067.055-.128.118-.19.18-.43.425-.854.855-1.286 1.275-.257.293-.687.365-1.026.17-.263-.132-.523-.264-.777-.413-1.115-.678-2.125-1.516-2.996-2.487-.808-.823-1.477-1.77-1.983-2.803-.256-.562-.2-.836.24-1.263.442-.427.915-.881 1.352-1.342.143-.128.225-.308.229-.5.003-.19-.074-.374-.212-.506-.053-.062-.109-.12-.164-.178L5.322 2.436c-.453-.444-.856-.449-1.31 0-.453.45-.88.845-1.293 1.294-.203.21-.367.453-.483.72-.18.574-.188 1.187-.021 1.765.2.897.517 1.764.94 2.58.728 1.44 1.65 2.773 2.742 3.964 1.878 2.126 4.255 3.76 6.92 4.753.606.228 1.244.361 1.89.396v.01z"/>
```
Comment (bocha) [inline styles/src/images/wharton/business_analytics/emeritus-phone-icon.svg:2]: This should be change from the CSS, not from here. Anyway, I added this fix in another PR, so is safe to remove this 🙂

File: styles/src/modules/wharton/business_analytics/sections/_case_studies.scss (hunk 1 above)
Comment (bocha) [reply styles/src/modules/wharton/business_analytics/sections/_case_studies.scss:11]: Agreed, we are already applying `1.5rem` for both sides of all sections on mobile. Just better do `margin: 5. 0;` here

File: styles/src/modules/wharton/business_analytics/sections/_certificate.scss [hunk 3]
```diff
@@ -1,4 +1,5 @@
 .certificate {
+  text-align: center;
```
Comment (bocha) [inline styles/src/modules/wharton/business_analytics/sections/_certificate.scss:2]: Empty line before next declaration is missing here.

File: styles/src/modules/wharton/business_analytics/sections/_certificate.scss [hunk 4]
```diff
@@ -11,4 +12,6 @@
 
     @include media-breakpoint-up(md) { margin: 0 0 3rem; }
   }
+
+  @include media-breakpoint-up(md) { text-align: left; }
```
Comment (bocha) [inline styles/src/modules/wharton/business_analytics/sections/_certificate.scss:16]: This should be above, just following the `text-align: center` from `.certificate`, to maintain consistency in declarations 😬

File: styles/src/modules/wharton/components/_hero.scss [hunk 5]
```diff
@@ -5,4 +5,9 @@
 
     @include media-breakpoint-up(lg) { background: $transparent; }
   }
+
+  &__form,
+  &__apply,
+  &__thank-you,
+  &__course-preview { padding: 2rem 2.5rem; }
```
Comment (bocha) [inline styles/src/modules/wharton/components/_hero.scss:12]: This was already fixed, the cause were overrides styles from EE, and not all share the same padding (e.g padding for `thank-you` is bigger than the one for the `cta`. So, please remove 🙂

File: styles/src/modules/wharton/hr_management_analytics/sections/_certificate.scss [hunk 6]
```diff
@@ -11,4 +12,6 @@
 
     @include media-breakpoint-up(md) { margin: 0 0 3rem; }
   }
+
+  @include media-breakpoint-up(md) { text-align: left; }
```
Comment (bocha) [inline styles/src/modules/wharton/hr_management_analytics/sections/_certificate.scss:16]: Same comments from above WBA `.certificate.scss` apply here.

**Pull request merged**